extensions/master_extension/v1/win-e2e-master-extension.sh

```diff
@@ -9,16 +9,24 @@ log() {
 	echo "$(date -R): $msg"
 }
 
+wait_for_agent_nodes() {
+	# upstream test requires at least two agent nodes
+	log "wait for agent nodes"
+	while [[ $(${KUBECTL} get nodes --selector=node-role.kubernetes.io/agent -ojson | jq '.items | length') -lt "2" ]]; do
```

Comment:
The gMSA tests have more than 2 nodes is it possible the extra nodes might get stuck as well?

Reply:
I updated `extensions/gmsa-coredns/v1/update-coredns.sh`, which should have the necessary logic to wait for the gmsa node to come online